src/pages/workspace/accounting/intacct/ExistingConnectionsPage.tsx

```diff
@@ -0,0 +1,64 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItemList from '@components/MenuItemList';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {getPoliciesConnectedToSageIntacct} from '@libs/actions/Policy/Policy';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import * as ReportUtils from '@libs/ReportUtils';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+
+type ExistingConnectionsPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.ACCOUNTING.EXISTING_SAGE_INTACCT_CONNECTIONS>;
+
+function ExistingConnectionsPage({route}: ExistingConnectionsPageProps) {
+    const {translate, datetimeToRelative} = useLocalize();
+    const styles = useThemeStyles();
+    const policiesConnectedToSageIntacct = getPoliciesConnectedToSageIntacct();
+    const policyID: string = route.params.policyID;
+
+    const menuItems = policiesConnectedToSageIntacct.map((policy) => {
+        const lastSuccessfulSyncDate = policy.connections?.intacct.lastSync?.successfulDate;
+        const date = lastSuccessfulSyncDate ? datetimeToRelative(lastSuccessfulSyncDate) : undefined;
+        return {
+            title: policy.name,
+            key: policy.id,
+            icon: policy.avatarURL ? policy.avatarURL : ReportUtils.getDefaultWorkspaceAvatar(policy.name),
+            iconType: policy.avatarURL ? CONST.ICON_TYPE_AVATAR : CONST.ICON_TYPE_WORKSPACE,
+            description: date ? translate('workspace.intacct.sageIntacctLastSync', date) : translate('workspace.accounting.intacct'),
+            onPress: () => {
+                // waiting for backend for reusing existing connections
+                Navigation.goBack(ROUTES.WORKSPACE_ACCOUNTING.getRoute(policyID));
+            },
+        };
+    });
+
+    return (
+        <ScreenWrapper
+            shouldEnablePickerAvoiding={false}
+            shouldShowOfflineIndicatorInWideScreen
+            testID={ExistingConnectionsPage.displayName}
+        >
+            <HeaderWithBackButton
+                title={translate('workspace.intacct.existingConnections')}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.goBack(ROUTES.WORKSPACE_ACCOUNTING.getRoute(policyID))}
+            />
+            <View style={[styles.flex1]}>
+                <MenuItemList
```

Comment:
From this issue https://github.com/Expensify/App/issues/59514, We forgot to add a ScrollView for the Existing connections list. If the list gets long enough to exceed the page length, it can’t be scrolled. context: https://github.com/Expensify/App/issues/59514#issuecomment-2771519209